Src/AutoFixture/RandomDateTimeOffsetSequenceGenerator.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Creates random <see cref="DateTimeOffset"/> specimens.
+    /// </summary>
+    /// <remarks>
+    /// The generated <see cref="DateTimeOffset"/> UTC values will be within
+    /// a range of ± two years from today's date,
+    /// unless a different range has been specified in the constructor.
+    /// </remarks>
+    public class RandomDateTimeOffsetSequenceGenerator : ISpecimenBuilder
+    {
+        private readonly RandomNumericSequenceGenerator _dateRandomizer;
```

Comment:
I've tried the change locally, and to be honest it add anything to the generator, but neither does it take away.

Do you want me to make the change to use the `RandomDateTimeSequenceGenerator`?